src/ccl/ccl.component.ts

```diff
@@ -3,16 +3,19 @@
  * App Component
  */
 import { Component, ViewEncapsulation } from '@angular/core';
+import { Sidebar } from './sidebar/sidebar.component';
+import { Chart } from './charts/chart.component';
```

Comment:
Maybe the syntax I referred to in my previous comment is some sugar that the ng-cli provides. That has a concept of bundles. If it's something we have to set up, then this is fine.